server/queries.js

```diff
@@ -64,7 +64,8 @@ const addSplit = async (req, res) => {
     ower_id: user.user_id, recipient_id: purchaserId, amount, trip_id: tripId, item_id: item.id,
   }));
   await Promise.all(userObjs.map((user) => SplitOwedPayment.create(user)));
-  users = users.map((user) => User.findByPk(user.user_id, { raw: true }));
+  // users = users.map((user) => User.findByPk(user.user_id, { raw: true }));
```

Comment:
remove

Reply:
This change has already been made in my most recent push